lib/websocket_server.rb

```diff
@@ -2,6 +2,8 @@ class WebsocketServer
   Pairing = Struct.new(:is_ws, :proxy)
 
   def initialize
+    @logger = Rails.env.production? ? $websocket_log : Logger.new(STDOUT)
```

Comment:
What if you run this worker via rake evm:start in development mode?  Where do STDOUT messages go?  Are they captured in the evm.log?